README.md

```diff
@@ -9,24 +9,48 @@ Branch Policy
 -------------
 
 - New development is done on **master** branch and most new change requests
-(PRs) should be proposed as changes to master, unless you know they are
-applicable to a particular release only.
-- Somewhere near a new major release, a numbered release branch (e.g. 14.0)
-is created from master, and the new branch goes into stabilization/release phase.
-- Major numbers are stepped up every new release.  The release schedule is
-driven by a time plan.  Use [semantic versioning](http://semver.org/) for
-minor and patch numbers.
-- GENIVI-specific release tags, such as "M-0.2" for the second baseline
-*pre*release for the "M" platform may also be added.
+  (PRs) should be proposed as changes to master, unless you know they are
+  applicable to a particular release only.
+- Major numbers are stepped up every new release of the GENIVI Compliance
+  Specification.  The release schedule is driven by a time plan. Use [semantic
+  versioning](http://semver.org/) for minor and patch numbers.
+- The GENIVI Compliance specification is released annually, while meta-ivi is
+  released bi-annually meaning that we get a gap. To keep the major version
+  numbers in sync, we will release 14.0.0 together with the Compliance
+  Specification and 14.50.0 half way to 15.0.0. 
+  - Note: Updates to the 14.0.0 release would then become 14.0.1 and/or 14.1.0, which
+    can be released __after__ 14.50.0.
```

Comment:
Change to "(Note that those could be released __after__ 14.50.0 in certain cases)"

Reply:
Fixed I think.